CONTRIBUTING.md

```diff
@@ -1,35 +1,55 @@
-# How to contribute
+# Symphony BDK for Java Contribution and Governance Policies
 
-If you found a bug or issue, please ensure the bug was not already reported by searching in
-[GitHub issues](https://github.com/SymphonyPlatformSolutions/symphony-api-client-java/issues).
-If you are unable to find an open issue addressing the problem, open a new one. 
-Be sure to include a title and clear description, the SDK version, and a code sample demonstrating the issue.
+This document describes the contribution process and governance policies of the FINOS {project name} project. The project is also governed by the [Linux Foundation Antitrust Policy](https://www.linuxfoundation.org/antitrust-policy/), and the FINOS [IP Policy](IP-Policy.pdf), [Code of Conduct](Code-of-Conduct.md), [Collaborative Principles](Collaborative-Principles.md), and [Meeting Procedures](Meeting-Procedures.md).
```

Comment:
Please review the links and put absolute paths.

Reply:
👍 ty for pointing that out, I fixed our project-blueprint accordingly.

```
This document describes the contribution process and governance policies of the FINOS {project name} project. The project is also governed by the [Linux Foundation Antitrust Policy](https://www.linuxfoundation.org/antitrust-policy/), and the FINOS [IP Policy](https://github.com/finos/community/blob/master/governance/IP-Policy.pdf), [Code of Conduct](https://github.com/finos/community/blob/master/governance/Code-of-Conduct.md), [Collaborative Principles](https://github.com/finos/community/blob/master/governance/Collaborative-Principles.md), and [Meeting Procedures](https://github.com/finos/community/blob/master/governance/Meeting-Procedures.md).
```

Reply:
Please replace all `{project name}` with `symphony-bdk` - assuming this is the name repository you want to give to the GitHub repository, after being transferred to github.com/finos

Reply:
`symphony-bdk-java` is OK? Assuming that `symphony-bdk-python` will follow later :)